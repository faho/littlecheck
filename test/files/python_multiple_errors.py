# RUN: /usr/bin/python %s
import sys

sys.stdout.write("foo\n")
# CHECK: foo
sys.stdout.write("one\n")
# CHECK: eins
sys.stdout.write("two\n")
# CHECK: zwei
sys.stdout.write("bar\n")
# CHECK: bar
