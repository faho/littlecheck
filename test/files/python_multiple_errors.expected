Failure in python_multiple_errors.py:

  The CHECK on line 7 wants:
    eins

  which failed to match line stdout:2:
    one

  Context:
    foo
    one <= does not match 'eins'
    two <= does not match 'zwei'
    bar

  when running command:
    /usr/bin/python python_multiple_errors.py
